src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -69,6 +70,17 @@ export default class StripViewModel extends BaseModel {
          */
         this._eventBus = EventBus;
 
+        /**
+         * If an aircraft is a Departure
+         *
+         * When this is true it means the aircraft is a Departure
+         * and if this is false it means the aircraft is an Arrival.
+         *
+         * @property isDeparture
+         * @type {boolean}
+         */
+        this.isDeparture = isDeparture;
```

Comment:
default this to `false`

Reply:
also, since the changeset doesn't extend down to the `._init()` method, add the assignment of `#isDeparture` in that method by using the passed `aircraftModel` reference.

Reply:
Fixed in latest commit